clients/java/src/test/java/io/camunda/zeebe/client/ZeebeClientTest.java

```diff
@@ -113,6 +116,21 @@ public void shouldOverridePropertyWithEnvVariable() {
     assertThat(builder.isPlaintextConnectionEnabled()).isFalse();
   }
 
+  @Test
+  public void shouldOverrideMaxMessageSizeWithEnvVar() {
+    // given
+    final ZeebeClientBuilderImpl builder = new ZeebeClientBuilderImpl();
+    builder.applyEnvironmentVariableOverrides(Boolean.TRUE);
+    builder.maxMessageSize(4 * ONE_MB);
+    Environment.system().put(MAX_MESSAGE_SIZE, "10MB");
+
+    // when
+    builder.build();
+
+    // then
+    assertThat(builder.getMaxMessageSize()).isEqualTo(10 * ONE_MB);
+  }
+
```

Comment:
🔧 We should aim to keep the test classes consistently structured. Please move this test case next to the other maxMessageSize test cases.

Reply:
fixed